Makefile

```diff
@@ -139,6 +139,10 @@ manifests: ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 	$(KUSTOMIZE) build config/default > config/render/capm3.yaml
 
+.PHONY: crd
```

Comment:
`Makefile` is imported from the upstream project, so I don't think it's ok to modify it here.